packages/jupyter-ai/style/pending-message.css

```diff
@@ -0,0 +1,12 @@
+.jp-ai-pending-message {
+  padding-right: 0;
+  color: var(--jp-ui-font-color2);
+}
+
+div.jp-ai-pending-message:not(:last-child) {
+  margin-bottom: 2em;
+}
+
+.jp-ai-pending-message p {
+  line-height: 0.6;
+}
```

Comment:
Can we define the styles inline using the `sx` prop on Material UI components? Doing so improves readability by keeping the component definition & styles in the same file.